### DISCUSSION TITLE
Web Streaming

### DISCUSSION THREAD

**Discussion by baqwas** [raspberrypi/picamera2#1155] [Q&A]
_Reference_: [Picamera Documentation](https://picamera.readthedocs.io/en/release-1.13/recipes2.html#rapid-capture-and-streaming)

_Help need_:
Using the example, I would like to craft a set of HTML files for viewing on a client browser. What would be the root folder used by the **http.server** module? Any guidance will be sincerely appreciated. Thanks.

Regards.

**Comment by GlassOnTin** [comment]
https://github.com/raspberrypi/picamera2/discussions/1166
